packages/xod-client/src/editor/components/CppImplementationEditor.jsx

```diff
@@ -15,11 +19,18 @@ const CppImplementationEditor = ({
   const options = {
     lineNumbers: true,
     readOnly: isInDebuggerTab,
-    mode: 'text/x-c++src',
+    mode: 'xod', // 'text/x-c++src',
```

Comment:
I guess we should call it `text/x-c++xod` to keep consistent. Eventually, we could invent yet more XOD-specific modes, like XOD Markdown doc.